samples/VSSample.Tests/HelloSequenceActivityTests.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Xunit;
```

Comment:
Please be sure to include the copyright header on all new files.

Reply:
Yes, of course. Thank you!